### ISSUE TITLE
Bulb stays lit when circuit shorts

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/circuit-construction-kit-common#492]
Test device:
Dell
Operating System:
Win 10
Browser:
chrome
Problem description:
For https://github.com/phetsims/QA/issues/311
First pointed out in a phethelp email.  When an ammeter is wired in parallel to a circuit to short it out, the lightbulb of the shorted circuit may remain lit.  In published version.
Steps to reproduce:
1. Build a simple circuit with a bulb, wires, and a battery
2. Add an ammeter parallel to the bulb

Screenshots:
![bugbulb](https://user-images.githubusercontent.com/41024075/56836284-02701a00-6834-11e9-9c63-df4a02d18411.png)

Troubleshooting information (do not edit):

<details>

Name: ‪Circuit Construction Kit: DC‬
URL: https://phet-dev.colorado.edu/html/circuit-construction-kit-dc/1.1.0-dev.3/phet/circuit-construction-kit-dc_en_phet.html
Version: 1.1.0-dev.3 2019-04-25 18:12:14 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.108 Safari/537.36
Language: en-US
Window: 1536x722
Pixel Ratio: 2.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by phet-steele** [comment]
Looks like a duplicate of #245?

**Comment by arouinfar** [comment]
Thanks @phet-steele! I knew there was as issue for this, but was searching only the closed issues. Oops!

@KatieWoe I moved your comment to the original issue.
